src/templates/Landing.js

```diff
@@ -16,7 +16,7 @@ import {
   DESKTOP_BREAKPOINT,
   ORANGE_PEEL_COLOR,
 } from '~/constants';
-import {mapNodeToProps, extract, track} from '~/utilities';
+import {mapNodeToProps, extract, track, classNames} from '~/utilities';
```

Comment:
Where is `classNames` used ?

Reply:
It isn't! Good catch. I wonder if Gatsby is intelligent enough to exclude it from the landing page bundle

Reply:
Cool just making sure that was the case.

Reply:
Once we remove this I will be good to approve